docs/hardhat-network/reference/README.md

```diff
@@ -0,0 +1,407 @@
+  - `0x14dC79964da2C08b23698B3D3cc7Ca32193d9955`
+  - `0x23618e81E3f5cdF7f54C3d65f7FBc0aBf5B21E8f`
+  - `0xa0Ee7A142d267C1f36714E4a8F75612F20a79720`
+  - `0xBcd4042DE499D14e55001CcbB24a551F3b954096`
+  - `0x71bE63f3384f5fb98995898A86B02Fb2426c5788`
+  - `0xFABB0ac9d68B0B445fB7357272Ff202C5651694a`
+  - `0x1CBd3b2770909D4e10f157cABC84C7264073C9Ec`
+  - `0xdF3e18d64BC6A983f673Ab319CCaE4f1a57C7097`
+  - `0xcd3B766CCDd6AE721141F452C550Ca635964ce71`
+  - `0x2546BcD3c84621e976D8185a91A922aE77ECEc30`
+  - `0xbDA5747bFD65F08deb54cb465eB87D40e51B197E`
+  - `0xdD2FD4581271e230360230F9337D5c0430Bf44C0`
+  - `0x8626f6940E2eb28930eFb4CeF49B2d1F2C9C1199`
+
+To customise it, take a look at [the configuration section](/config/README.md#hardhat-network).
+
+## JSON-RPC methods support
+
+### Standard methods
+
+#### `eth_accounts`
+
+#### `eth_blockNumber`
+
+#### `eth_call`
+
+#### `eth_chainId`
+
+#### `eth_coinbase`
+
+#### `eth_estimateGas`
+
+#### `eth_gasPrice`
+
+#### `eth_getBalance`
+
+#### `eth_getBlockByHash`
+
+#### `eth_getBlockByNumber`
+
+#### `eth_getBlockTransactionCountByHash`
+
+#### `eth_getBlockTransactionCountByNumber`
+
+#### `eth_getCode`
+
+#### `eth_getFilterChanges`
+
+#### `eth_getFilterLogs`
+
+#### `eth_getLogs`
+
+#### `eth_getStorageAt`
+
+#### `eth_getTransactionByBlockHashAndIndex`
+
+#### `eth_getTransactionByBlockNumberAndIndex`
+
+#### `eth_getTransactionByHash`
+
+#### `eth_getTransactionCount`
+
+#### `eth_getTransactionReceipt`
+
+#### `eth_mining`
+
+#### `eth_newBlockFilter`
+
+#### `eth_newFilter`
+
+#### `eth_newPendingTransactionFilter`
+
+#### `eth_pendingTransactions`
+
+#### `eth_sendRawTransaction`
+
+#### `eth_sendTransaction`
+
+#### `eth_sign`
+
+#### `eth_signTypedData_v4`
+
+#### `eth_subscribe`
+
+#### `eth_syncing`
+
+#### `eth_uninstallFilter`
+
+#### `eth_unsubscribe`
+
+#### `net_listening`
+
+#### `net_peerCount`
+
+#### `net_version`
+
+#### `web3_clientVersion`
+
+#### `web3_sha3`
+
+### Hardhat network methods
+
+#### `hardhat_addCompilationResult`
+
+Add information about compiled contracts
+
+#### `hardhat_dropTransaction`
+
+Remove a transaction from the mempool
+
+#### `hardhat_getStackTraceFailuresCount`
+
+#### `hardhat_impersonateAccount`
+
+see the [Mainnet Forking guide](../../guides/mainnet-forking.md)
+
+#### `hardhat_intervalMine`
+
+#### `hardhat_reset`
+
+see the [Mainnet Forking guide](../../guides/mainnet-forking.md)
+
+#### `hardhat_setBalance`
+
+Modifies the balance of an account.
+
+#### `hardhat_setCode`
+
+Modifies the code of an account.
+
+#### `hardhat_setLoggingEnabled`
+
+Enable or disable logging in Hardhat Network
+
+#### `hardhat_setMinGasPrice`
+
+change the minimum gas price accepted by the network (in wei)
+
+#### `hardhat_setNonce`
+
+Modifies an account's nonce by overwriting it. Throws an InvalidInputError if nonce is smaller than the current one. The reason for this restriction is to avoid collisions when deploying contracts using the same nonce more than once.
```

Comment:
```suggestion
Modifies an account's nonce by overwriting it. Throws an `InvalidInputError` if nonce is smaller than the current one. The reason for this restriction is to avoid collisions when deploying contracts using the same nonce more than once.
```